rules-tests/CodeQuality/Rector/Assign/CombinedAssignRector/config/configured_rule.php

```diff
@@ -3,10 +3,14 @@
 declare(strict_types=1);
 
 use Rector\CodeQuality\Rector\Assign\CombinedAssignRector;
+use Rector\Core\Configuration\Option;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set(Option::AUTOLOAD_PATHS, [__DIR__ . '/autoloader.php']);
```

Comment:
I did more testing.. I guess the whole point of my issue is, that neither `Option::AUTOLOAD_PATHS` nor `Option::OPTION_AUTOLOAD_FILE` include the autoloader in a way that rector makes use of it.

it does not gets triggered when files cannot be found at runtime.

adding a manual `require_once __DIR__ . '/autoloader.php'';` inside my rector rule makes the autoloader work like expected

Reply:
I see. The includes files were moved to `BOOTSTRAP_FILES`:

I've updated the release to include it: https://github.com/rectorphp/rector/releases/tag/0.10.0

> adding a manual require_once __DIR__ . '/autoloader.php''; inside my rector rule makes the autoloader work like expected. In your case:

```php
$parameters->set(Option::BOOTSTRAP_FILES, [
    __DIR__ . '/autoload.php',
]);
```


Thanks for feedback

Reply:
Ref https://github.com/rectorphp/getrector.org/commit/03661372fbbb9e6711446fad1df424a0b49cc0b7

Reply:
thx for the feedback. it doesn't even work with `Option::BOOTSTRAP_FILES`

Reply:
That's weird :thinking: I'll try to run this PR locally

Reply:
When I run tests, it's passing :+1: 

```bash
vendor/bin/phpunit rules-tests/CodeQuality/Rector/Assign/CombinedAssignRector/CombinedAssignRectorTest.php
```

![image](https://user-images.githubusercontent.com/924196/115987560-d53adb80-a5b5-11eb-9e0a-882c5432ef71.png)

Reply:
you can see the actual problem in the class not found error, inside your screenshot

Reply:
Foudn it... the tests were missing autoload/bootstrap feature

Reply:
See https://github.com/rectorphp/rector/pull/6239/files